app/assets/javascripts/ckeditor/loader.js.erb

```diff
@@ -1,3 +1,4 @@
 //= require ckeditor/init
 
 CKEDITOR.config.customConfig = "<%= javascript_path "ckeditor/config.js" %>";
+CKEDITOR.addCss(".cke_editable { background-color: #f9f9f9 }");
```

Comment:
Isn't it possible vo use CSS for this? :thinking:

Reply:
All the documentation I looked at was solved this way. Although I seem to remember that there was some way to load a css and apply those changes(but I think it was always load through the js). I can investigate it if necessary.